src/commands/install-node.yml

```diff
@@ -59,11 +59,27 @@ steps:
         - run:
             name: Install and use node version
             command: |
-              # if("<<parameters.version>>" -eq "latest") {
-              #   choco install nodejs
-              # } else {
-              #   $r = Invoke-WebRequest https://nodejs.org/download/release/index.json | ConvertFrom-Json
-              #   $latest_of_version = ($r | where { $_.version -like "v<<parameters.version>>*" })[0].version.substring(1)
-              #   choco install nodejs --version $latest_of_version --allow-downgrade
-              # }
+              $nodeIndex = "https://nodejs.org/download/release/index.json"
+
+              if ("<<parameters.version>>" -eq "latest") {
+                choco install nodejs
+              } elseif ("<<parameters.version>>" -eq "lts") {
+                $registry =  (Invoke-WebRequest $nodeIndex).content
+                $lts = $registry |
+                    jq -c -r 'first(.[] | select(.lts != false)).version | ltrimstr(\"v\")'
```

Comment:
Use `jq` to iterate over the registry until it finds the first object in which `lts != false` (this will always be the latest LTS version), then access the `.version` field and trim `v` from `v14.16.6`.

jq is pre-installed on the windows image: https://circleci.com/docs/2.0/hello-world-windows/#software-pre-installed-in-the-windows-image

Reply:
Nice 😁 
![image](https://user-images.githubusercontent.com/1084688/134971022-17480dc7-0665-428b-83f0-e70ad5d65f45.png)

Reply:
Nice! 😁 
![image](https://user-images.githubusercontent.com/1084688/134972052-de2c1d35-459d-4c1e-9f73-7edf22a6ba9b.png)